content/gsoc/gsoc-2024/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "GSoC 2024"
+date: ""
+---
+
+* * *
+
+### We are applying again to be a mentor organization for GSoC in 2024
+
+This will be the 15th time that Honeynet is participating in the Google Summer of Code program to accelerate the creation of information- and cyber security related tools that are free and open source software. Thank you to all mentors who submitted their project ideas and will volunteer as mentors throughout the summer.
+
+On this page you will learn more about GSoC 2024 and our efforts on developing and improving our projects.
+
+### Project Ideas
+(Will Update Soon)
+
+Often students ask us what kind of projects we are likely to be offering in a coming year. If you want to get an idea of the kind of projects ideas we proposed and the accepted projects we ran during GSoC 2009-2023, you can view these here:
+
+- [GSoC 2023 Project Ideas](https://www.honeynet.org/gsoc/gsoc-2023/google-summer-of-code-2023-project-ideas/)
+- [GSoC 2022 Project Ideas](https://www.honeynet.org/gsoc/gsoc-2022/google-summer-of-code-2022-project-ideas/)
+- [GSoC 2021 Project Ideas](https://www.honeynet.org/gsoc/gsoc-2021/google-summer-of-code-2021-project-ideas/)  
+- [GSoC 2020 Project Ideas](https://www.honeynet.org/gsoc/gsoc-2020/google-summer-of-code-2020-project-ideas/)  
+- [GSoC 2018 Project Ideas](//www.honeynet.org/gsoc/gsoc-2018/ideas/)
+- [GSoC 2017 Project Ideas](/gsoc/gsoc-2017/ideas/)
+- [GSoC 2016 Project Ideas](/gsoc/gsoc-2016/ideas/)
+- [GSoC 2015 Project Ideas](/gsoc/gsoc-2015/ideas/)
+- [GSoC 2014 Project Ideas](/gsoc/gsoc-2014/ideas/)  
+- [GSoC 2013 Project Ideas](/gsoc/gsoc-2013/ideas/)
+- [GSoC 2012 Project Ideas](/gsoc/gsoc-2012/ideas/)
+- [GSoC 2011 Project Ideas](/gsoc/gsoc-2011/ideas/)
+- [GSoC 2010 Project Ideas](/gsoc/gsoc-2010/ideas/)
+- [GSoC 2009 Project Ideas](/gsoc/gsoc-2009/ideas/)
+
+### Contacting Us
+
+If you have any questions, please drop by in our [Slack chat](//gsoc-slack.honeynet.org/) and say "hi" (note that our members do have to sleep and come from timezones all over the world, so please be patient and wait a few hours if you have a specific question).
+
+[![GSoC Slack](//gsoc-slack.honeynet.org/badge.svg)](//gsoc-slack.honeynet.org/)
+
+### Organizational Administrators
+
+Will update soon.
+
+### Why get involved with the Honeynet Project?
```

Comment:
This section, including the links needs an overhaul/review.
If you can fix the links we can merge and I can help with the overhaul.

Reply:
Hey sorry for the delay,
Can you please elaborate which links should I add ?